script.js

```diff
@@ -1,4 +1,183 @@
+2 - Display results. Player one Swaps if needed
+3 - Player two Swaps if needed
+4 - Converting array into actual numbers, then choosing winner
+*/
+
+/*Game types:
+"a" - Highest number wins. Default mode.
+"b" - Lowerst number wins
+*/
+
+//Generate a pair of numbers for player one. Push into playerOneRolls[] array
+//Generate a pair of numbers for player two. Push into playerTwoRolls[] array
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  console.log(`>>>>Running game.`);
+  console.log(`- Current mode: ${currentMode}.`);
+  if (input == "") {
+    var myOutputValue = `Hi! You get two dice each. After rolling, we will put both dice side-by-side to see who comes up with the biggest numerical combination!<br/>Each player can choose the order in which the dice are placed.<br/>To roll, please enter "Roll"!`;
+  }
+  if (currentMode == 4 && gameType == "b" && input == "") {
+    var myOutputValue = comparingSmallestNumbers();
+  }
+
+  if (currentMode == 4 && gameType == "a" && input == "") {
+    var myOutputValue = comparingBiggestNumbers();
+  }
+  if (currentMode == 3) {
+    var myOutputValue = playerTwoShifting(input); //Mode changed to 4 in this function
+  }
+  if (currentMode == 2) {
+    var myOutputValue = playerOneShifting(input); //Mode changed to 3 in this function
+  }
+  if (input == "LCN" && currentMode == 1) {
+    gameType = "b";
+    console.log(`Game type: ${gameType}`);
+    return `LCN mode set! Now, enter "Roll" to begin!.`;
+  }
+  if (input == "Go again" || (input == "Roll" && currentMode == 1)) {
+    currentMode == 1;
+    playerOneRolls = [];
+    playerTwoRolls = [];
+    rollingPlayerOne();
+    rollingPlayerTwo();
+    currentMode = 2;
+    var myOutputValue = `Player one's numbers: ${playerOneRolls}<br/>Player two's numbers: ${playerTwoRolls}<br/><br/>Now, decide whether you wish to Swap the numbers, or wish to keep them as is!<br/><br/>Player one, please decide first.<br/>Type in "Swap" and click submit to swap the numbers, or click the submit button to keep the number order.`;
+    console.log(`- Current mode: ${currentMode}. Game type: ${gameType}.`);
+  }
+  return myOutputValue;
+};
+
+//P1 numbers rolling
+var rollingPlayerOne = function () {
+  counter = 0;
+  while (counter < 2) {
+    var rNG = randomNumberGenerator();
+    playerOneRolls.push(rNG); //Stores numbers
+    console.log(`Rolling P1 number. P1 number(s) pushed: ${playerOneRolls}`);
+    counter += 1;
+  }
+  playerOneNumber = playerOneRolls.join(""); //Joins the two numbers, pushes into playerOneNumber[] array
+};
+
+//P2 numbers rolling
+var rollingPlayerTwo = function () {
+  counter = 0;
+  while (counter < 2) {
+    var rNG = randomNumberGenerator();
+    playerTwoRolls.push(rNG); //Stores numbers
+    console.log(`Rolling P2 number. P2 number(s) pushed: ${playerTwoRolls}`);
+    counter += 1;
+  }
+  playerTwoNumber = playerTwoRolls.join(""); //Joins the two numbers, pushes into playerOneNumber[] array
+};
```

Comment:
If you see that you have functions that are doing similar things it's usually a sign that you can combine it into 1 function and use parameters to be able to reuse the function.  This would help if you have more than 2 players in the future. 

Here's how the function might look and will be used
```
//Numbers rolling function
var rollingPerPlayer = function (playerArray) {
  counter = 0;
  while (counter < 2) {
    var rNG = randomNumberGenerator();
    playerArray.push(rNG); //Stores numbers
    counter += 1;
  }
  playerOneNumber = playerArray.join(""); //Joins the two numbers, pushes into playerOneNumber[] array
  // return
  return playerOneNumber;
};

// using the function
rollingPerPlayer(playerOneRolls)
rollingPerPlayer(playerTwoRolls)
```